backend/alembic/versions/c0472125ea51_invite_developers.py

```diff
@@ -0,0 +1,44 @@
+"""invite_developers
+
+Revision ID: c0472125ea51
+Revises: da1ce859085a
+Create Date: 2023-08-07 19:37:20.008567
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = 'c0472125ea51'
+down_revision = 'da1ce859085a'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('directuploadappinvite',
+    sa.Column('id', sa.Integer(), nullable=False),
+    sa.Column('app_id', sa.Integer(), nullable=False),
+    sa.Column('developer_id', sa.Integer(), nullable=False),
+    sa.Column('is_primary', sa.Boolean(), nullable=False),
```

Comment:
I found the inclusion of this field surprising. Is this really used/needed? I feel like that design might be something we would want to talk about. I at least can't come up with a sane one.

I think making somebody to be the primary should be a separate action and happen instantly, without needing the receiving part to accept.

Reply:
Yeah, I intended for that to be used to invite someone transfer primary ownership, but didn't implement it yet. It should probably be a separate action, but I'm not sure being able to dump maintainership on someone without their approval is a good idea.

Reply:
Let's remove the field and do it later?